core/templates/pages/exploration-editor-page/services/exploration-rights.service.ts

```diff
@@ -247,6 +261,14 @@ export class ExplorationRightsService {
           response.rights.community_owned,
           response.rights.viewable_if_private
         );
+      })
+      .catch(error => {
```

Comment:
Just a thought: now that the frontend doesn't actually reflect the state of the backend, I think we need to show some confirmation to the user. Can you show a green toast in the bottom-left/bottom-right on successful deletion saying something like "Successfully removed owner/editor/viewer: {{username}}"? (I think we might already have some functionality for this sort of thing, so I'd suggest reusing it.)

Reply:
Should the confirmation show after the frontend updates or after the backend updates?

Reply:
Done